src/ui/components/SecondaryToolbox/ReactDevTools.tsx

```diff
@@ -260,20 +280,43 @@ function createReactDevTools(
 // We can work around this by checking RD's "bridge protocol" version (which we also store)
 // and loading the appropriate frontend version to match.
 // For more information see https://github.com/facebook/react/issues/24219
-async function loadReactDevToolsInlineModuleFromProtocol(stateUpdaterCallback: Function) {
-  const response = await ThreadFront.evaluate({
+async function loadReactDevToolsInlineModuleFromProtocol(
+  stateUpdaterCallback: Function,
+  replayClient: ReplayClientInterface,
+  pauseId?: string
+) {
+  if (!pauseId) {
+    return;
+  }
+
+  const response = await ThreadFront.evaluateNew({
     asyncIndex: 0,
     text: ` __RECORD_REPLAY_REACT_DEVTOOLS_SEND_MESSAGE__("getBridgeProtocol", undefined)`,
   });
-  const json: any = await response?.returned?.getJSON();
-  const version = json?.data?.version;
-
-  // We should only load the DevTools module once we know which protocol version it requires.
-  // If we don't have a version yet, it probably means we're too early in the Replay session.
-  if (version >= 2) {
-    stateUpdaterCallback(await import("react-devtools-inline/frontend"));
-  } else if (version === 1) {
-    stateUpdaterCallback(await import("react-devtools-inline_4_17_0/frontend"));
+  if (response.returned?.object) {
+    // Unwrap the nested eval objects by asking the backend for contents
+    // of the nested fields: `{data: {version: 123}}`
+    const responseDataObject = await getObjectPropertyHelper(
```

Comment:
:speech_balloon: Now that I've got `getJSON()`, I _could_ change this to be one `getJSON()` call instead and access `obj.data.version`. I may do that as a follow-up.